exhibitor-core/src/main/java/com/netflix/exhibitor/core/HttpsConfiguration.java

```diff
@@ -0,0 +1,197 @@
+            return this;
+        }
+
+        public Builder truststorePath(String truststorePath)
+        {
+            httpsConf = new HttpsConfiguration(httpsConf.serverKeystorePath, httpsConf.serverKeystorePassword,
+                httpsConf.clientKeystorePath, httpsConf.clientKeystorePassword, truststorePath,
+                httpsConf.truststorePassword, httpsConf.requireClientCert, httpsConf.verifyPeerCert);
+            return this;
+        }
+
+        public Builder truststorePassword(String truststorePassword)
+        {
+            httpsConf = new HttpsConfiguration(httpsConf.serverKeystorePath, httpsConf.serverKeystorePassword,
+                httpsConf.clientKeystorePath, httpsConf.clientKeystorePassword, httpsConf.truststorePath,
+                truststorePassword, httpsConf.requireClientCert, httpsConf.verifyPeerCert);
+            return this;
+        }
+
+        public Builder requireClientCert(String requireClientCertString)
+        {
+            boolean requireClientCertValue;
+            if ( requireClientCertString != null )
+            {
+                requireClientCertValue = Boolean.parseBoolean(requireClientCertString);
+            }
+            else
+            {
+                requireClientCertValue = httpsConf.verifyPeerCert;
+            }
+
+            return requireClientCert(requireClientCertValue);
+        }
+
+        public Builder requireClientCert(boolean requireClientCert)
+        {
+            httpsConf = new HttpsConfiguration(httpsConf.serverKeystorePath, httpsConf.serverKeystorePassword,
+                httpsConf.clientKeystorePath, httpsConf.clientKeystorePassword, httpsConf.truststorePath,
+                httpsConf.truststorePassword, requireClientCert, httpsConf.verifyPeerCert);
+            return this;
+        }
+
+        public Builder verifyPeerCert(String verifyPeerCertString)
+        {
+            boolean verifyPeerCertValue;
+            if ( verifyPeerCertString != null )
+            {
+                verifyPeerCertValue = Boolean.parseBoolean(verifyPeerCertString);
+            }
+            else
+            {
+                verifyPeerCertValue = httpsConf.verifyPeerCert;
+            }
+
+            return verifyPeerCert(verifyPeerCertValue);
+        }
+
+        public Builder verifyPeerCert(boolean verifyPeerCert)
+        {
+            httpsConf = new HttpsConfiguration(httpsConf.serverKeystorePath, httpsConf.serverKeystorePassword,
+                httpsConf.clientKeystorePath, httpsConf.clientKeystorePassword, httpsConf.truststorePath,
+                httpsConf.truststorePassword, httpsConf.requireClientCert, verifyPeerCert);
+            return this;
+        }
+
+        public HttpsConfiguration build()
+        {
+            Preconditions.checkArgument(!(httpsConf.serverKeystorePath == null ^ httpsConf.serverKeystorePassword == null),
+                "Server keystore path/password must be both/neither specified.");
```

Comment:
@rdeliallisi and @gpaul can you work together to make these error messages be super helpful and correct English at the same time? :)

Reply:
The simplest error message would be:
```
Both or neither of EXHIBITOR_TLS_SERVER_KEYSTORE_PATH and EXHIBITOR_TLS_SERVER_KEYSTORE_PASSWORD must be specified.
```
I'd be happy to accept that change.

The same goes for the truststore check.